jmx/jmx.go

```diff
@@ -121,10 +121,10 @@ func Close() {
 	}
 
 	cancel()
-	_ = cmdIn.Close()
-	_ = cmdError.Close()
 
 	done.Wait()
+	_ = cmdIn.Close()
```

Comment:
What will happen if the jmx query takes more than timeout to execute?
https://golang.org/pkg/sync/#Cond.Wait